client/replays/replay-drop.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useEffect } from 'react'
+import { useAppDispatch } from '../redux-hooks'
+import { openReplay } from './action-creators'
+
+export default function ReplayDrop() {
+  const dispatch = useAppDispatch()
+
+  const onDrop = useCallback(
+    (e: DragEvent) => {
+      e.preventDefault()
+      e.stopPropagation()
```

Comment:
Calling `stopPropagation` is an anti-pattern and I'd avoid using it unless you *really* need to.